decidim_app-design/app/views/public/drafts/draft-view-access-requested.html.erb

```diff
@@ -0,0 +1,112 @@
+<%
+@title = "Borrador -- Decidim Barcelona"
+@activesection = "drafts"
+@activepage = "drafts"
+%>
+
+<%= partial "template_top" %>
+
+<main>
+  <div class="wrapper">
+    <%= partial "process_header" %>
+    <div class="row">
+      <div class="columns section view-side mediumlarge-4 mediumlarge-push-8 large-3 large-push-9">
+        <%= partial "followme" %>
+        <div class="card text-center">
+          <div class="card__content">
+            <div>
+              <strong class="text-large text-uppercase">version <%= Faker::Number.between(1, 50) %></strong>
+            </div>
+            <div>
+              <span class="text-medium"><u>see other versions</u></span>
+            </div>
+            <button class="button secondary light expanded button--sc mt-s"><%= icon "check", class: "icon--small" %>&nbsp;Acceso solicitado</button>
+          </div>
+          <div class="card__status">
+            <ul class="card-data">
+              <li class="card-data__item">
```

Comment:
Great! thanks for adding the anchor so that your layout is closer to the finally required tags :clap: